### ISSUE TITLE
NullReferenceException when adding asset via GenericUnityObjects

### ISSUE THREAD

**Issue by kpietraszko** [SolidAlloy/ClassTypeReference-for-Unity#30]
NRE seems to be caused because you're trying to access Event.current from outside OnGUI. Here's the stacktrace:
```
NullReferenceException: Object reference not set to an instance of an object
TypeReferences.Editor.TypeDropdown.SelectionTree.set_SelectedNode (TypeReferences.Editor.TypeDropdown.SelectionNode value) (at Library/PackageCache/com.solidalloy.type-references@2.11.1/Editor/TypeDropdown/SelectionTree.cs:65)
TypeReferences.Editor.TypeDropdown.SelectionTree.SetSelection (TypeReferences.Editor.TypeDropdown.TypeItem[] items, System.Type selectedType) (at Library/PackageCache/com.solidalloy.type-references@2.11.1/Editor/TypeDropdown/SelectionTree.cs:124)
TypeReferences.Editor.TypeDropdown.SelectionTree..ctor (TypeReferences.Editor.TypeDropdown.TypeItem[] items, System.Type selectedType, System.Action`1[T] onTypeSelected, System.Int32 searchbarMinItemsCount, System.Boolean hideNoneElement) (at Library/PackageCache/com.solidalloy.type-references@2.11.1/Editor/TypeDropdown/SelectionTree.cs:50)
GenericUnityObjects.Editor.ScriptableObjects.SelectionWindow.OneTypeSelectionWindow.GetSelectionTree (GenericUnityObjects.Editor.Util.NonGenericAttribute attribute, System.Action`1[T] onTypeSelected) (at Library/PackageCache/com.solidalloy.generic-unity-objects@2.11.1/Editor/ScriptableObjects/SelectionWindow/OneTypeSelectionWindow.cs:32)
GenericUnityObjects.Editor.ScriptableObjects.SelectionWindow.OneTypeSelectionWindow.OnCreate (System.Action`1[T] onTypeSelected, System.String[] genericArgNames, System.Type[][] genericParamConstraints) (at Library/PackageCache/com.solidalloy.generic-unity-objects@2.11.1/Editor/ScriptableObjects/SelectionWindow/OneTypeSelectionWindow.cs:22)
GenericUnityObjects.Editor.ScriptableObjects.TypeSelectionWindow.Create (System.Type genericTypeWithoutArgs, System.Action`1[T] onTypesSelected) (at Library/PackageCache/com.solidalloy.generic-unity-objects@2.11.1/Editor/ScriptableObjects/TypeSelectionWindow.cs:37)
GenericUnityObjects.Editor.ScriptableObjects.GenericSOCreator.CreateAsset (System.Type genericTypeWithoutArgs, System.String fileName) (at Library/PackageCache/com.solidalloy.generic-unity-objects@2.11.1/Editor/ScriptableObjects/GenericSOCreator.cs:33)
```
I'm using Unity 2020.3.19f1 and version 2.11.1 of your packages

**Comment by SolidAlloy** [comment]
Hi, thanks for the heads up. It was a bad idea to put `Event.current.Use()` in the `SelectedNode` setter :P 
Fixed in version **2.11.2** of both packages

**Comment by kpietraszko** [comment]
Wow that was fast. Thanks!
